<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Temperatura</class>
 <widget class="QDialog" name="Temperatura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperatura</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/icono</normaloff>:/icono</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Centigrados</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdCent">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialCent">
        <property name="minimum">
         <number>-273</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Fahrenheit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="lcdFar"/>
      </item>
      <item>
       <widget class="QDial" name="dialFar">
        <property name="minimum">
         <number>-460</number>
        </property>
        <property name="maximum">
         <number>210</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Kelvin</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
      <item>
       <widget class="QDial" name="dialKel">
        <property name="maximum">
         <number>372</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialFar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFar</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
